### DISCUSSION TITLE
translation updates ("bulk attributions" tab)

### DISCUSSION THREAD

**Discussion by abubelinha** [bionomia/bionomia#376] [General]
I know there is something called Crowdin for translation purposes. 
I don't know how it works, but I don't plan to make translations so I am a bit lazy to learn that. 
I just want to comment small fixes if I find translation bugs which can confuse people.  
Is it OK to comment them here, or is Crowdin also the preferred way of submitting them?

In this particular case ("bulk attributions" tab), '_A file containing at most 10,000 records is generated_' was originally translated as '_A file containing at **least** 10,000 records **was** generated_'. 
I correct that and add some other minor improvements (include "you" as "usted" -which can be omitted in Spanish- in order to avoid confusion -without it, the verb can refer to the person or to the file-):

> Download discovered specimen records, work offline, then upload new claims as a csv. A file containing **at most** 10,000 records **is generated**. If **you have many more than this**, you may work in batches. This routine does not update underlying specimen data nor does it update existing claims. If existing claims are present in the upload, these will be skipped. Your uploaded csv must use a comma to separate columns each optionally enclosed by quotes.

**suggested corrections** in Spanish version:

> Descargue registros de especímenes descubiertos, trabaje desconectado, luego cargue adjudicaciones nuevas como csv. **Se genera** un archivo que contiene **como máximo** 10,000 registros. Si **usted tiene muchos más que estos**, puede trabajar en tandas. Esta rutina no actualiza los datos de los especímenes subyacentes ni actualiza las adjudicaciones existentes. Si hay reclamos existentes en el archivo a cargar, estos se omitirán. Su csv cargado debe usar una coma para separar las columnas, cada una opcionalmente entre comillas.

**Comment by dshorthouse** [comment]
Thanks for this. I added this to the Crowdin translations. Will be deployed sometime this week.
